docs/source/supercheq/Supercheq_qss.ipynb

```diff
@@ -5,73 +5,113 @@
    "id": "7769945b",
```

Comment:
<p>Put the buttons in the same line if possible</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/541/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I'm not sure how to get them in the same line. Even if I put the code in the same markdown cell, the buttons appear below one another.</p>

Reply:
Strange... They look right in https://github.com/Infleqtion/client-superstaq/pull/551 so maybe just wait until those changes are merged into main? Or copy and paste what Pranav had and maybe that'll fix it